OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -476,32 +481,31 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </note>
                                 </item>
                                 <item xml:id="e4">
-                                    <locus target="#26"/>
-                                    <note>A green thread for navigating is inserted on the upper corner of the folio.</note> <!-- move this to binding -->
-                                </item>
-                                <item xml:id="e5">
                                     <note>Text written on erasure, e.g. <locus target="#12ra #39rb #44rb"/>.</note>
                                 </item>
-                                <item xml:id="e6">                         
+                                <item xml:id="e5">                         
                                     <note>Erasures marked with thin black lines, e.g. <locus target="#73vb #74va"/>.</note>
                                 </item>
                             </list>
                         </additions>
                         
                         <bindingDesc>
                             <binding contemporary="true">
-                                <decoNote xml:id="b1">Two wooden boards covered with reddish-brown tooled leather.
-                                    <!-- two layers of leather cover!! -->
+                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards covered with reddish-brown <material key="leather"/>.
+                                    Presence of a reddish-brown tooled <material key="leather"/> overback in correspondence of the spine.
```

Comment:
"The same leather also appears on the spine" I would make more discursive in view of catalogue. Also, should we note that the leather only covers about 40% of the boards?

Reply:
That additional level of cover along the spine side (and parts of the boards) should be the so-called "overback". Is that the case @SusanneHummel ? This overback issue asks for some discussion, because it is not clear in the guidelines how to deal with this (not so frequent) binding feature.

Reply:
I haven't checked the images, but I think, Massimo, you're right. See also the expl. in the ligatus lexicon: https://www.ligatus.org.uk/lob/search?search_api_views_fulltext=overback

And also here: https://www.ligatus.org.uk/sites/default/files/manual20050110.pdf